test/e2e/install_maximal_and_replay_test.go

```diff
@@ -0,0 +1,139 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package e2e
+
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ It would be nice to have a comment included with the test that states the expected behavior. I had forgotten that re-running with new variables didn't update the config. This also confused a couple testers when I had introduced the OPW installer.

Reply:
I can run a description before the test, think that can help